test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/CalculatorTests.cs

```diff
@@ -192,4 +193,29 @@ public void OverUtilized()
         Assert.Equal(1500UL, record.MemoryUsedInBytes);
         Assert.Equal(1000UL, record.SystemResources.GuaranteedMemoryInBytes);
     }
+
+    /// <summary>
+    /// Ensure that CPU stats work appropriately.
+    /// </summary>
+    [Fact]
+    public void DoubleCpuUnits_WithFullyUtilizaed()
+    {
+        var limitedResources = new SystemResources(2, 2, TotalMemoryInBytes, TotalMemoryInBytes);
```

Comment:
Please add parameter names for opaque values
```suggestion
        var limitedResources = new SystemResources(???: 2, ???: 2, TotalMemoryInBytes, TotalMemoryInBytes);
```

Also, consider writing initialisers as follows as it improves readability:
```suggestion
        SystemResources limitedResources = new(2, 2, TotalMemoryInBytes, TotalMemoryInBytes);
```